src/components/FeaturePanel/Property.tsx

```diff
@@ -34,12 +34,16 @@ const Value = styled.div`
 `;
 
 const DefaultRenderer = ({ v }) => v;
-const renderers = {
+var renderers = {
   website: WebsiteRenderer,
   phone: PhoneRenderer,
   opening_hours: OpeningHoursRenderer,
 };
 
+renderers["contact:website"] = WebsiteRenderer;
+renderers["contact:phone"] = PhoneRenderer;
+renderers["contact:mobile"] = PhoneRenderer;
+
 const Property = ({ k, v, onEdit }) => {
   const Renderer = renderers[k] || DefaultRenderer;
 
```

Comment:
Also you have to add these two tags in `featuredTags` - here:
https://github.com/zbycz/osmapp/blob/51d8dc6/src/components/FeaturePanel/FeaturePanel.tsx#L55

I agree this part of application has dirty code, it was the first code written in 2017. 
After we merge this, i can add some refactoring.

Reply:
That's the bit I missed, thanks